sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/AppConfigurationRefresh.java

```diff
@@ -173,10 +173,8 @@ private boolean refreshStores() {
                     }
                 }
             } catch (Exception e) {
-                if (refreshInterval != null) {
-                    // The next refresh will happen sooner if refresh interval is expired.
-                    StateHolder.resetAll(refreshInterval, appProperties);
-                }
+                // The next refresh will happen sooner if refresh interval is expired.
+                StateHolder.resetAll(refreshInterval, appProperties);
```

Comment:
The name `resetAll` is a bit confusing. It sounds like you want to "reset" everything to original default state (like clear refresh attempts). But actually you are calculating backoff times and incrementing refresh attempts. How about calling this `UpdateNextRefreshTime`?